ui/src/app/components/resources.tsx

```diff
@@ -54,6 +57,7 @@ export const ResourceCardMenu: React.FunctionComponent<{
         }]
       )
     }
+    disabled={resourceType !== ResourceType.CONCEPT_SET || !canEdit}
```

Comment:
Won't this be always disabled for something not a concept set? This should maybe be an &&?

Reply:
oh, snap, you're right! thank you!